packages/backend-tasks/api-report.md

```diff
@@ -11,7 +11,11 @@ import { Logger } from 'winston';
 
 // @public
 export interface PluginTaskScheduler {
+  createScheduledLocalTaskRunner(schedule: TaskScheduleDefinition): TaskRunner;
```

Comment:
Seeing as these are very similar to the existing ones and I want to make sure we explore some options, what do you think about adding a `location` or something like that as an option to `TaskScheduleDefinition` instead? I'm not sure which one I like better but figured we could consider it x)

As it is at the moment I'm feeling we need to call this a breaking change, seeing as we're adding new required methods to an interface that the backend might be providing. Making it an option is a bit of a workaround to that too.

Reply:
Yeah not a bad idea at all. There's the small risk that one might introduce future  parameters that only make sense in one of the locations - but yeah, I think that your suggestion is pretty safe and should make things slightly cleaner.

Reply:
(Not gonna call it `location` though for sure :P 🚲 🏠)